aqua/BGHUDAppKit/Portfile

```diff
@@ -25,6 +25,14 @@ if {[vercmp $xcodeversion 5] >=0} {
     patchfiles-append xcode5.patch
 }
 
+# This is a temporary kludge. The new Xcode build system fails to
+# build and destroot this for reasons that are poorly understood.
+# Remove this when a better fix is known.
+if {[vercmp ${xcodeversion} 10.0] >= 0} {
+    build.pre_args      -UseNewBuildSystem=NO
```

Comment:
This is probably not needed for build, only for destroot. If it is needed for build, it means that you probably have broken `#include`s and those should get fixed instead.

Reply:
It is indeed needed also for build. I will provide a build.log shortly...

Reply:
It may very well be that this is caused by wrong header includes. However, I can't see any helpful hint on that in the output.

Right now I can't upload files, so...
```
version:1
:debug:sysinfo macOS 10.14 (darwin/18.0.0) arch i386
:debug:sysinfo MacPorts 2.5.4
:debug:sysinfo Xcode 10.0
:debug:sysinfo SDK 10.14
:debug:sysinfo MACOSX_DEPLOYMENT_TARGET: 10.14
:debug:main Executing org.macports.main (BGHUDAppKit)
:debug:main dropping privileges: euid changed to 503, egid changed to 501.
:debug:archivefetch archivefetch phase started at Fri Oct  5 17:01:39 CEST 2018
:debug:archivefetch Executing org.macports.archivefetch (BGHUDAppKit)
version:1
:debug:sysinfo macOS 10.14 (darwin/18.0.0) arch i386
:debug:sysinfo MacPorts 2.5.4
:debug:sysinfo Xcode 10.0
:debug:sysinfo SDK 10.14
:debug:sysinfo MACOSX_DEPLOYMENT_TARGET: 10.14
:debug:main Executing org.macports.main (BGHUDAppKit)
:debug:main dropping privileges: euid changed to 503, egid changed to 501.
:debug:fetch fetch phase started at Fri Oct  5 17:01:39 CEST 2018
:notice:fetch --->  Fetching distfiles for BGHUDAppKit
:debug:fetch Executing org.macports.fetch (BGHUDAppKit)
:debug:fetch Privilege de-escalation not attempted as not running as root.
:debug:checksum checksum phase started at Fri Oct  5 17:01:39 CEST 2018
:notice:checksum --->  Verifying checksums for BGHUDAppKit
:debug:checksum Executing org.macports.checksum (BGHUDAppKit)
:info:checksum --->  Checksumming BGHUDAppKit-0.7.tar.gz
:debug:checksum Calculated (rmd160) is 3a26a8497eff103774e86bcd04c1c61b2601faa6
:debug:checksum Correct (rmd160) checksum for BGHUDAppKit-0.7.tar.gz
:debug:checksum Calculated (sha256) is e0f89ecb2692c53a3144ce6f2fcd8b9c2cb814d52a16ed2cc8f5b15b7e87ea52
:debug:checksum Correct (sha256) checksum for BGHUDAppKit-0.7.tar.gz
:debug:checksum Privilege de-escalation not attempted as not running as root.
:debug:extract extract phase started at Fri Oct  5 17:01:39 CEST 2018
:notice:extract --->  Extracting BGHUDAppKit
:debug:extract Executing org.macports.extract (BGHUDAppKit)
:info:extract --->  Extracting BGHUDAppKit-0.7.tar.gz
:debug:extract setting option extract.args to '/opt/local/var/macports/distfiles/BGHUDAppKit/BGHUDAppKit-0.7.tar.gz'
:debug:extract Environment: 
:debug:extract CC_PRINT_OPTIONS='YES'
:debug:extract CC_PRINT_OPTIONS_FILE='/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/.CC_PRINT_OPTIONS'
:debug:extract CPATH='/opt/local/include'
:debug:extract LIBRARY_PATH='/opt/local/lib'
:debug:extract MACOSX_DEPLOYMENT_TARGET='10.14'
:info:extract Executing:  cd "/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work" && /usr/bin/gzip -dc '/opt/local/var/macports/distfiles/BGHUDAppKit/BGHUDAppKit-0.7.tar.gz' | /usr/bin/tar -xf - 
:debug:extract system:  cd "/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work" && /usr/bin/gzip -dc '/opt/local/var/macports/distfiles/BGHUDAppKit/BGHUDAppKit-0.7.tar.gz' | /usr/bin/tar -xf - 
:debug:extract euid/egid changed to: 0/0
:debug:extract chowned /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work to macports
:debug:extract euid/egid changed to: 503/501
:debug:extract Executing proc-post-org.macports.extract-extract-0
:debug:extract Privilege de-escalation not attempted as not running as root.
:debug:patch patch phase started at Fri Oct  5 17:01:40 CEST 2018
:debug:patch Executing org.macports.patch (BGHUDAppKit)
:notice:patch --->  Applying patches to BGHUDAppKit
:info:patch --->  Applying xcode5.patch
:debug:patch Environment: 
:debug:patch CC_PRINT_OPTIONS='YES'
:debug:patch CC_PRINT_OPTIONS_FILE='/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/.CC_PRINT_OPTIONS'
:debug:patch CPATH='/opt/local/include'
:debug:patch LIBRARY_PATH='/opt/local/lib'
:debug:patch MACOSX_DEPLOYMENT_TARGET='10.14'
:info:patch Executing:  cd "/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7" && /usr/bin/patch -p0 < '/Users/lass/ports/aqua/BGHUDAppKit/files/xcode5.patch'
:debug:patch system:  cd "/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7" && /usr/bin/patch -p0 < '/Users/lass/ports/aqua/BGHUDAppKit/files/xcode5.patch'
:info:patch patching file BGHUDAppKit.xcodeproj/project.pbxproj
:debug:patch Privilege de-escalation not attempted as not running as root.
:debug:configure configure phase started at Fri Oct  5 17:01:40 CEST 2018
:notice:configure --->  Configuring BGHUDAppKit
:debug:configure Preferred compilers: clang macports-clang-5.0 macports-clang-4.0
:debug:configure Using compiler 'Xcode Clang'
:debug:configure Executing org.macports.configure (BGHUDAppKit)
:debug:configure Privilege de-escalation not attempted as not running as root.
:debug:build build phase started at Fri Oct  5 17:01:40 CEST 2018
:notice:build --->  Building BGHUDAppKit
:debug:build Executing org.macports.build (BGHUDAppKit)
:debug:build Assembled command: 'cd "/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7" && /usr/bin/xcodebuild  -alltargets -configuration Deployment build INSTALL_PATH=/opt/local/Library/Frameworks  OBJROOT="/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/" SYMROOT="/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/" MACOSX_DEPLOYMENT_TARGET=10.14 ARCHS=x86_64 SDKROOT="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk" GCC_VERSION=com.apple.compilers.llvm.clang.1_0 CLANG_CXX_LIBRARY="libc++" '
:debug:build system: cd "/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7" && /usr/bin/xcodebuild  -alltargets -configuration Deployment build INSTALL_PATH=/opt/local/Library/Frameworks  OBJROOT="/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/" SYMROOT="/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/" MACOSX_DEPLOYMENT_TARGET=10.14 ARCHS=x86_64 SDKROOT="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk" GCC_VERSION=com.apple.compilers.llvm.clang.1_0 CLANG_CXX_LIBRARY="libc++" 
:info:build Build settings from command line:
:info:build     ARCHS = x86_64
:info:build     CLANG_CXX_LIBRARY = libc++
:info:build     GCC_VERSION = com.apple.compilers.llvm.clang.1_0
:info:build     INSTALL_PATH = /opt/local/Library/Frameworks
:info:build     MACOSX_DEPLOYMENT_TARGET = 10.14
:info:build     OBJROOT = /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/
:info:build     SDKROOT = /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk
:info:build     SYMROOT = /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/
:info:build 2018-10-05 17:01:41.281 xcodebuild[9001:8729103] [MT] IDELogStore: Failed to open log store at /opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Build
:info:build 2018-10-05 17:01:41.281 xcodebuild[9001:8729103] [MT] IDELogStore: Failed to open Build log store: Error Domain=IDEFoundationErrorDomain Code=1 "Unable to create log store directory at '/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Build': (513) You don’t have permission to save the file “Build” in the folder “Logs”." UserInfo={NSLocalizedDescription=Unable to create log store directory at '/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Build': (513) You don’t have permission to save the file “Build” in the folder “Logs”.}. User info: {
:info:build     NSLocalizedDescription = "Unable to create log store directory at '/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Build': (513) You don\U2019t have permission to save the file \U201cBuild\U201d in the folder \U201cLogs\U201d.";
:info:build }.
:info:build 2018-10-05 17:01:41.291 xcodebuild[9001:8729103] [MT] IDELogStore: Failed to open log store at /opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Test
:info:build 2018-10-05 17:01:41.291 xcodebuild[9001:8729103] [MT] IDELogStore: Failed to open Test log store: Error Domain=IDEFoundationErrorDomain Code=1 "Unable to create log store directory at '/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Test': (513) You don’t have permission to save the file “Test” in the folder “Logs”." UserInfo={NSLocalizedDescription=Unable to create log store directory at '/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Test': (513) You don’t have permission to save the file “Test” in the folder “Logs”.}. User info: {
:info:build     NSLocalizedDescription = "Unable to create log store directory at '/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb/Logs/Test': (513) You don\U2019t have permission to save the file \U201cTest\U201d in the folder \U201cLogs\U201d.";
:info:build }.
:info:build --- xcodebuild: WARNING: Configuration Deployment is not in the project. Building default configuration.
:info:build 2018-10-05 17:01:41.804 xcodebuild[9001:8729103] [MT] DVTAssertions: Warning in /Library/Caches/com.apple.xbs/Sources/IDEFrameworks_Fall2018/IDEFrameworks-14320.25/IDEFoundation/Buildables/IDEWorkspaceArena.m:1146
:info:build Details:  Failed to create workspace arena at <DVTFilePath:0x7fcc517d2f00:'/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb'>: Error Domain=NSCocoaErrorDomain Code=513 "You don’t have permission to save the file “BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb” in the folder “DerivedData”." UserInfo={NSFilePath=/opt/local/var/macports/home/Library/Developer/Xcode/DerivedData/BGHUDAppKit-ckvgiwnwtekeahcuchaeexdvdgdb, NSUnderlyingError=0x7fcc517997a0 {Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted"}}
:info:build Object:   <IDEWorkspaceArena: 0x7fcc5175e100>
:info:build Method:   -createWorkspaceArenaFolderIfNecessary
:info:build Thread:   <NSThread: 0x7fcc4ed06460>{number = 1, name = main}
:info:build Please file a bug at https://bugreport.apple.com with this warning message and any useful information you can provide.
:info:build note: Using new build system
:info:build note: Planning build
:info:build note: Constructing build description
:info:build Build system information
:info:build warning: Traditional headermap style is no longer supported; please migrate to using separate headermaps and set 'ALWAYS_SEARCH_USER_PATHS' to NO. (in target 'BGHUDAppKitPlugin')
:info:build Build system information
:info:build warning: Traditional headermap style is no longer supported; please migrate to using separate headermaps and set 'ALWAYS_SEARCH_USER_PATHS' to NO. (in target 'BGHUDAppKit')
:info:build CreateBuildDirectory /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build (in target: BGHUDAppKitPlugin)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     builtin-create-build-directory /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin (in target: BGHUDAppKitPlugin)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin/Contents (in target: BGHUDAppKitPlugin)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin/Contents
:info:build SymLink /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources Versions/Current/Resources (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/ln -sfh Versions/Current/Resources /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources
:info:build error: unable to create symlink at '/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources'
:info:build SymLink /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/Current A (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/ln -sfh A /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/Current
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin/Contents/MacOS (in target: BGHUDAppKitPlugin)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin/Contents/MacOS
:info:build SymLink /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Headers Versions/Current/Headers (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/ln -sfh Versions/Current/Headers /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Headers
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin/Contents/Resources (in target: BGHUDAppKitPlugin)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources/BGHUDAppKitPlugin.ibplugin/Contents/Resources
:info:build SymLink /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/BGHUDAppKit Versions/Current/BGHUDAppKit (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/ln -sfh Versions/Current/BGHUDAppKit /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/BGHUDAppKit
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/A/Resources (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/A/Resources
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/A/Headers (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/A/Headers
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/A (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions/A
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Versions
:info:build MkDir /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework (in target: BGHUDAppKit)
:info:build     cd /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7
:info:build     /bin/mkdir -p /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework
:info:build ** BUILD FAILED **
:info:build The following build commands failed:
:info:build 	SymLink /opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/Release/BGHUDAppKit.framework/Resources Versions/Current/Resources
:info:build (1 failure)
:info:build Command failed: cd "/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7" && /usr/bin/xcodebuild  -alltargets -configuration Deployment build INSTALL_PATH=/opt/local/Library/Frameworks  OBJROOT="/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/" SYMROOT="/opt/local/var/macports/build/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/work/BGHUDAppKit-0.7/build/" MACOSX_DEPLOYMENT_TARGET=10.14 ARCHS=x86_64 SDKROOT="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk" GCC_VERSION=com.apple.compilers.llvm.clang.1_0 CLANG_CXX_LIBRARY="libc++" 
:info:build Exit code: 65
:error:build Failed to build BGHUDAppKit: command execution failed
:debug:build Error code: NONE
:debug:build Backtrace: command execution failed
:debug:build     while executing
:debug:build "$procedure $targetname"
:error:build See /opt/local/var/macports/logs/_Users_lass_ports_aqua_BGHUDAppKit/BGHUDAppKit/main.log for details.
```

Reply:
Okay, that's new and special. I've not seen _that_ failure before. Clearly you're right and you do need it for build as well. This is the first case I've seen.